.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Relevant Issues
+
+<!--
+It is highly recommended to link to an issue that has been approved by a maintainer, indicated by an approved label.
+Not doing so may result in the pr being closed, without a reason.
```

Comment:
I don't quite like this phrasing, we ought be kinder here.

> We highly recommend linking to an issue that has been approved by a maintainer, or making one yourself before opening a PR. Otherwise it may not be clear why a chance is necessary or you might do work that we'd already decided not to implement that is not immediately obvious, leading to wasted time and effort.